src/fxcorrelator_xengops.py

```diff
@@ -298,9 +300,9 @@ def print_vacc_statuses(vstatus):
     corr.logger.info('\tx engines have vacc ld time %i' % xldtime)
 
     # wait for the vaccs to arm
-    corr.logger.info('\twaiting %i seconds for accumulations to start' % wait_time)
+    wait_time = corr.time_from_mcnt(ldmcnt) - time.time() + 0.2
```

Comment:
Including tabs in log strings can confuse tools that parse the output. Marc has complained about this :)